CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 
 All notable changes to the Wazuh app project will be documented in this file.
+## Wazuh v4.2.0 - Kibana 7.10.0 , 7.10.2 - Revision 4201
+
+### Fixed
```

Comment:
Maybe this should be `Changed` instead of `Fixed`.

Reply:
I have change it. Thanks you!